docs/section-metadata.md

```diff
@@ -206,6 +207,25 @@ Adds a gradient overlay over the section.
 
 ![gradient-secondary-blur-overlay-example.png](assets/gradient-secondary-blur-overlay-example.png)
 
+#### Highlight
+
+Adds background color to the section and cancels padding. Available in two variants, _highlight_ and _highlight-medium_.
+
+**_Example Highlight:_**
+
+| Section Metadata |                                                           |
+|------------------|-----------------------------------------------------------|
+| Style            | highlight                                                 |
+| Background-Image | ![columns-subnavigation.png](assets/columns-subnavigation.png) |
```

Comment:
Rather than `Background-Image` property in the section metadata, shouldn't this just be an example image?